lib/exhal/navigation.ex

```diff
@@ -184,13 +180,41 @@ defmodule ExHal.Navigation do
     end
   end
 
+  defp _follow_links(client, [link], tmpl_vars, opts) do
+    [_follow_link(client, link, tmpl_vars, opts)]
+  end
+
+  defp _follow_links(client, links, tmpl_vars, opts) do
+    {:ok, tsup} = Task.Supervisor.start_link()
+
+    try do
+      Task.Supervisor.async_stream_nolink(tsup, links, &_follow_link(client, &1, tmpl_vars, opts))
+      |> Enum.map(fn
+        {:ok, followed_link} -> followed_link
+        {:exit, reason} -> raise inspect(reason)
+      end)
+    after
+      Supervisor.stop(tsup, :normal)
+    end
+  end
+
   defp _follow_link(client, link, tmpl_vars, opts) do
     cond do
       Link.embedded?(link) ->
         {:ok, link.target, %ResponseHeader{status_code: 200}}
 
-      :else ->
+      true ->
```

Comment:
:(

Reply:
I was confused at how that worked... Oh... `:else` is truthy. I'll switch it back. I had never seen it before.